src/template.js

```diff
@@ -530,11 +534,13 @@ module.exports = {
             'p-rep-email-address': repEmailAddress.route,
             'p-rep-telephone-number': repTelephoneNumber.route,
             'p-rep-reference-number': repReferenceNumber.route,
-            'p-rep-claims-management-reg': repClaimsManagementRegNumber.route
+            'p-rep-claims-management-reg': repClaimsManagementRegNumber.route,
+            'p--new-or-existing-application': newOrExistingApp.route,
+            'p--contact-cica': contactCica.route
         }
     },
     answers: {},
-    progress: ['p-applicant-fatal-claim'],
+    progress: ['p--new-or-existing-application'],
```

Comment:
and here change to p-applicant-fatal-claim

Reply:
Same as above, this is now the first page in the application, so it needs to replace 'p-applicant-fatal-claim' here.

Reply:
ah of course¬